bundles/org.openhab.binding.renault/src/main/resources/OH-INF/i18n/renault.properties

```diff
@@ -55,6 +55,9 @@ thing-type.config.renault.car.vin.description = Vehicle Identification Number
 # channel types
 
 channel-type.renault.batteryavailableenergy.label = Battery Energy Available
+channel-type.renault.batterystatusupdated.label = Battery Status Updated
+channel-type.renault.batterystatusupdated.description = Timestamp of the last battery status update
+channel-type.renault.batterystatusupdated.state.pattern = %1$tH:%1$tM %1$td.%1$tm.%1$tY
```

Comment:
I added the English and reverted the French.  I think this is how it is documented to be done.